build/x86_64_centos7/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM centos:7
+
+COPY puias-computational-x86_64.repo /etc/yum.repos.d/
+RUN rpm --import http://springdale.math.ias.edu/data/puias/7/x86_64/os/RPM-GPG-KEY-puias \
```

Comment:
springdale.math.ias.edu, eh? Could we add some commentary to explain? Thanks.

Reply:
This is literally the first hit for ["centos7 install boost 1.55"](https://www.google.com/search?q=centos7+install+boost+1.55):
https://centos.pkgs.org/7/puias-computational-x86_64/boost155-1.55.0-12.sdl7.x86_64.rpm.html

That said, it may be ok for local development (i.e., to put in the README), but probably too flaky for an official build Dockerfile. Let me think on this — we may decide to build boost from source instead.

Reply:
Thought more about it. Let's get the Xenial build container in first — I'll re-add the CentOS 7 one in a separate PR.